<?xml version="1.0" ?>
<!-- This file was generated by robot assembler -->
<robot name="FIXED_ARM"
       xmlns:xi="http://www.w3.org/2001/XInclude"
       xmlns:gazebo="http://playerstage.sourceforge.net/gazebo/xmlschema/#gz"
       xmlns:model="http://playerstage.sourceforge.net/gazebo/xmlschema/#model"
       xmlns:sensor="http://playerstage.sourceforge.net/gazebo/xmlschema/#sensor"
       xmlns:body="http://playerstage.sourceforge.net/gazebo/xmlschema/#body"
       xmlns:geom="http://playerstage.sourceforge.net/gazebo/xmlschema/#geom"
       xmlns:joint="http://playerstage.sourceforge.net/gazebo/xmlschema/#joint"
       xmlns:interface="http://playerstage.sourceforge.net/gazebo/xmlschema/#interface"
       xmlns:rendering="http://playerstage.sourceforge.net/gazebo/xmlschema/#rendering"
       xmlns:renderable="http://playerstage.sourceforge.net/gazebo/xmlschema/#renderable"
       xmlns:controller="http://playerstage.sourceforge.net/gazebo/xmlschema/#controller"
       xmlns:physics="http://playerstage.sourceforge.net/gazebo/xmlschema/#physics"
       xmlns:xacro="http://www.ros.org/wiki/xacro">
  <xacro:macro name="gazebo_link_reference" params="link_name mu1:=0.9 mu2:=0.9" >
    <gazebo reference="${link_name}">
      <mu1>${mu1}</mu1>
      <mu2>${mu2}</mu2>
    </gazebo>
  </xacro:macro>
  <xacro:macro name="gazebo_transmission" params="joint_name" >
    <transmission name="${joint_name}_trans">
      <type>transmission_interface/SimpleTransmission</type>
      <joint name="${joint_name}">
        <hardwareInterface>hardware_interface/VelocityJointInterface</hardwareInterface>
      </joint>
      <actuator name="${joint_name}_motor">
        <hardwareInterface>hardware_interface/VelocityJointInterface</hardwareInterface>
        <mechanicalReduction>1</mechanicalReduction>
      </actuator>
    </transmission>
  </xacro:macro>

  <xacro:gazebo_link_reference link_name="BASE" />
  <link name="BASE">
  <inertial>
     <mass value="100.12"/>
     <origin rpy="0 0 0" xyz="-1.443271e-19 1.438274e-05 0.002031"/>
     <inertia ixx="10.0001" ixy="1.731926e-19" ixz="4.433066e-19" iyy="10.0001" iyz="-3.739513e-05" izz="10.0"/>
  </inertial>
  <visual>
     <origin xyz="0.000000 0.000000 0.000000" rpy="0.00000000 -0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/foot-plate.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.000000 0.000000 0.000000" rpy="0.00000000 -0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/foot-plate.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="0.000000 0.000000 0.004000" rpy="3.14159265 -0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_s103_a.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.000000 0.000000 0.004000" rpy="3.14159265 -0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_s103_a.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="-0.000000 0.000000 0.029500" rpy="0.00000000 0.00000000 -3.14159265"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="-0.000000 0.000000 0.029500" rpy="0.00000000 0.00000000 -3.14159265"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  </link>
  <xacro:gazebo_link_reference link_name="LINK0" />
  <link name="LINK0">
  <inertial>
     <mass value="0.015"/>
     <origin rpy="0 0 0" xyz="-1.953584e-17 3.484759e-19 0.01455"/>
     <inertia ixx="1.969763e-06" ixy="-6.618208e-22" ixz="-1.400051e-21" iyy="3.159763e-06" iyz="6.539791e-23" izz="3.012500e-06"/>
  </inertial>
  <visual>
     <origin xyz="-0.000000 -0.000000 0.028000" rpy="-1.57079633 0.00000000 -1.57079633"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_h101.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="-0.000000 -0.000000 0.028000" rpy="-1.57079633 0.00000000 -1.57079633"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_h101.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  </link>
  <xacro:gazebo_link_reference link_name="LINK1" />
  <link name="LINK1">
  <inertial>
     <mass value="0.228"/>
     <origin rpy="0 0 0" xyz="-0.000658 -0.012202 0.003807"/>
     <inertia ixx="0.000153" ixy="3.026316e-08" ixz="2.519013e-06" iyy="0.000136" iyz="-8.800526e-07" izz="6.231353e-05"/>
  </inertial>
  <visual>
     <origin xyz="-0.000000 -0.000000 -0.019000" rpy="0.00000000 -0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="-0.000000 -0.000000 -0.019000" rpy="0.00000000 -0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="-0.000000 -0.000000 -0.016000" rpy="1.57079633 0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_x102.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="-0.000000 -0.000000 -0.016000" rpy="1.57079633 0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_x102.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="-0.000000 0.000000 0.020600" rpy="-1.03831454 -1.57079633 1.03831454"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="-0.000000 0.000000 0.020600" rpy="-1.03831454 -1.57079633 1.03831454"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="-0.000000 0.000000 0.034850" rpy="1.57079633 -0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/hr12_f101.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="-0.000000 0.000000 0.034850" rpy="1.57079633 -0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/hr12_f101.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  </link>
  <xacro:gazebo_link_reference link_name="LINK2" />
  <link name="LINK2">
  <inertial>
     <mass value="0.138"/>
     <origin rpy="0 0 0" xyz="-3.260870e-05 -0.084159 -0.017913"/>
     <inertia ixx="0.000131" ixy="6.371739e-08" ixz="-4.891304e-09" iyy="2.835168e-05" iyz="2.098587e-06" izz="0.000124"/>
  </inertial>
  <visual>
     <origin xyz="0.000000 0.000000 -0.019000" rpy="0.00000000 1.57079633 -3.14159265"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_h103.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.000000 0.000000 -0.019000" rpy="0.00000000 1.57079633 -3.14159265"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_h103.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="0.000000 -0.055000 -0.019000" rpy="-1.57079633 0.00000000 -0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_x103.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.000000 -0.055000 -0.019000" rpy="-1.57079633 0.00000000 -0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_x103.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="0.000000 -0.110150 -0.019000" rpy="-0.00000000 0.00000000 3.14159265"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.000000 -0.110150 -0.019000" rpy="-0.00000000 0.00000000 3.14159265"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  </link>
  <xacro:gazebo_link_reference link_name="LINK3" />
  <link name="LINK3">
  <inertial>
     <mass value="0.035"/>
     <origin rpy="0 0 0" xyz="2.956365e-17 0.037793 -0.019"/>
     <inertia ixx="2.398520e-05" ixy="-2.377463e-20" ixz="-3.875419e-21" iyy="7.029167e-06" iyz="5.681579e-21" izz="2.676187e-05"/>
  </inertial>
  <visual>
     <origin xyz="0.000000 0.000000 -0.019000" rpy="0.07059318 1.57079633 0.07059318"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_h101.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.000000 0.000000 -0.019000" rpy="0.07059318 1.57079633 0.07059318"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_h101.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="0.000000 0.083000 -0.019000" rpy="2.87954722 1.57079633 -0.26204543"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_h103.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.000000 0.083000 -0.019000" rpy="2.87954722 1.57079633 -0.26204543"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_h103.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  </link>
  <xacro:gazebo_link_reference link_name="LINK4" />
  <link name="LINK4">
  <inertial>
     <mass value="0.228"/>
     <origin rpy="0 0 0" xyz="0.016793 -0.012202 -0.018342"/>
     <inertia ixx="6.231353e-05" ixy="8.800526e-07" ixz="2.519013e-06" iyy="0.000136" iyz="-3.026316e-08" izz="0.000153"/>
  </inertial>
  <visual>
     <origin xyz="0.000000 0.000000 -0.019000" rpy="0.00000000 -0.00000000 -0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.000000 0.000000 -0.019000" rpy="0.00000000 -0.00000000 -0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="-0.014250 0.000000 -0.019000" rpy="1.83704838 -1.57079633 -0.26625205"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/hr12_f101.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="-0.014250 0.000000 -0.019000" rpy="1.83704838 -1.57079633 -0.26625205"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/hr12_f101.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="0.036600 0.000000 -0.019000" rpy="-2.11121583 -1.57079633 -2.60117315"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_x102.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.036600 0.000000 -0.019000" rpy="-2.11121583 -1.57079633 -2.60117315"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_x102.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="0.039600 0.000000 -0.019000" rpy="2.61807145 -1.57079633 -2.61807145"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.039600 0.000000 -0.019000" rpy="2.61807145 -1.57079633 -2.61807145"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  </link>
  <xacro:gazebo_link_reference link_name="LINK5" />
  <link name="LINK5">
  <inertial>
     <mass value="0.135"/>
     <origin rpy="0 0 0" xyz="-8.085089e-17 0.040606 0.009833"/>
     <inertia ixx="5.790463e-05" ixy="9.869592e-20" ixz="3.170078e-21" iyy="3.497750e-05" iyz="4.888000e-06" izz="4.397463e-05"/>
  </inertial>
  <visual>
     <origin xyz="0.000000 0.000000 0.020500" rpy="0.44051066 1.57079633 0.44051066"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_h101s.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.000000 0.000000 0.020500" rpy="0.44051066 1.57079633 0.44051066"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_h101s.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="-0.000000 0.020000 0.020500" rpy="1.57079633 -0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_s103_a.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="-0.000000 0.020000 0.020500" rpy="1.57079633 -0.00000000 0.00000000"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_s103_a.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="-0.000000 0.045500 0.020500" rpy="1.57079633 0.00000000 -3.14159265"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="-0.000000 0.045500 0.020500" rpy="1.57079633 0.00000000 -3.14159265"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  </link>
  <xacro:gazebo_link_reference link_name="LINK6" />
  <link name="LINK6">
  <inertial>
     <mass value="0.148"/>
     <origin rpy="0 0 0" xyz="0.00748 -0.000689 0.025936"/>
     <inertia ixx="7.515967e-05" ixy="4.610676e-07" ixz="1.693519e-05" iyy="8.733643e-05" iyz="-6.564527e-07" izz="5.476682e-05"/>
  </inertial>
  <visual>
     <origin xyz="0.024000 0.000000 0.019500" rpy="-3.14159265 0.00000000 1.57079633"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_s101.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.024000 0.000000 0.019500" rpy="-3.14159265 0.00000000 1.57079633"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_s101.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="0.024000 0.000000 0.019500" rpy="-0.39234014 -1.57079633 -1.17845618"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.024000 0.000000 0.019500" rpy="-0.39234014 -1.57079633 -1.17845618"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/xm430_dynamixel.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  <visual>
     <origin xyz="0.003000 -0.000000 0.019500" rpy="1.57079633 -0.00000000 -1.57079633"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_e170.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.003000 -0.000000 0.019500" rpy="1.57079633 -0.00000000 -1.57079633"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_e170.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  </link>
  <xacro:gazebo_link_reference link_name="LINK7" />
  <link name="LINK7">
  <inertial>
     <mass value="0.035"/>
     <origin rpy="0 0 0" xyz="0.023397 0.018542 -0.019"/>
     <inertia ixx="1.643493e-05" ixy="-8.139500e-06" ixz="-5.964431e-21" iyy="2.583362e-05" iyz="-1.010962e-21" izz="2.525856e-05"/>
  </inertial>
  <visual>
     <origin xyz="0.000000 0.000000 -0.019000" rpy="0.38930895 1.57079633 0.91290772"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_e171.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </visual>
  <collision>
     <origin xyz="0.000000 0.000000 -0.019000" rpy="0.38930895 1.57079633 0.91290772"/>
     <geometry>
     <mesh filename="package://robot_assembler/meshes/fr12_e171.dae" scale="1.000000 1.000000 1.000000"/>
     </geometry>
  </collision>
  </link>
  <xacro:gazebo_transmission joint_name="JOINT0" />
  <joint name="JOINT0" type="revolute">
     <parent link="BASE"/>
     <child link="LINK0"/>
     <origin xyz="-0.000000 0.000000 0.048500" rpy="0.00000000 0.00000000 -3.14159265"/>
     <axis xyz="0 0 1"/>
     <limit effort="200" lower="-3.14159" upper="3.14159" velocity="40"/>
  </joint>
  <xacro:gazebo_transmission joint_name="JOINT1" />
  <joint name="JOINT1" type="revolute">
     <parent link="LINK0"/>
     <child link="LINK1"/>
     <origin xyz="-0.000000 -0.020500 0.028000" rpy="-1.57079633 -0.00000000 0.00000000"/>
     <axis xyz="0 0 1"/>
     <limit effort="200" lower="-3.14159" upper="3.14159" velocity="40"/>
  </joint>
  <xacro:gazebo_transmission joint_name="JOINT2" />
  <joint name="JOINT2" type="revolute">
     <parent link="LINK1"/>
     <child link="LINK2"/>
     <origin xyz="-0.019000 -0.000000 0.020600" rpy="-1.03831454 -1.57079633 1.03831454"/>
     <axis xyz="0 0 1"/>
     <limit effort="200" lower="-3.14159" upper="3.14159" velocity="40"/>
  </joint>
  <xacro:gazebo_transmission joint_name="JOINT3" />
  <joint name="JOINT3" type="revolute">
     <parent link="LINK2"/>
     <child link="LINK3"/>
     <origin xyz="0.000000 -0.110150 -0.000000" rpy="-0.00000000 0.00000000 3.14159265"/>
     <axis xyz="0 0 1"/>
     <limit effort="200" lower="-3.14159" upper="3.14159" velocity="40"/>
  </joint>
  <xacro:gazebo_transmission joint_name="JOINT4" />
  <joint name="JOINT4" type="revolute">
     <parent link="LINK3"/>
     <child link="LINK4"/>
     <origin xyz="0.000000 0.083000 -0.000000" rpy="-0.00000000 -0.00000000 0.00000000"/>
     <axis xyz="0 0 1"/>
     <limit effort="200" lower="-3.14159" upper="3.14159" velocity="40"/>
  </joint>
  <xacro:gazebo_transmission joint_name="JOINT5" />
  <joint name="JOINT5" type="revolute">
     <parent link="LINK4"/>
     <child link="LINK5"/>
     <origin xyz="0.020600 0.000000 -0.019000" rpy="2.61807145 -1.57079633 -2.61807145"/>
     <axis xyz="0 0 1"/>
     <limit effort="200" lower="-3.14159" upper="3.14159" velocity="40"/>
  </joint>
  <xacro:gazebo_transmission joint_name="JOINT6" />
  <joint name="JOINT6" type="revolute">
     <parent link="LINK5"/>
     <child link="LINK6"/>
     <origin xyz="-0.000000 0.064500 0.020500" rpy="1.57079633 0.00000000 -3.14159265"/>
     <axis xyz="0 0 1"/>
     <limit effort="200" lower="-3.14159" upper="3.14159" velocity="40"/>
  </joint>
  <xacro:gazebo_transmission joint_name="JOINT7" />
  <joint name="JOINT7" type="revolute">
     <parent link="LINK6"/>
     <child link="LINK7"/>
     <origin xyz="0.024000 0.019000 0.019500" rpy="-0.39234014 -1.57079633 -1.17845618"/>
     <axis xyz="0 0 1"/>
     <limit effort="200" lower="-3.14159" upper="3.14159" velocity="40"/>
  </joint>

  <!-- Gazebo plugin for ROS Control -->
  <gazebo>
    <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
      <robotNamespace>/</robotNamespace>
      <robotSimType>gazebo_ros_control/DefaultRobotHWSim</robotSimType>
    </plugin>
  </gazebo>
</robot>
